main.go

```diff
@@ -184,11 +185,10 @@ func getTimeout() time.Duration {
 func getHeartbeatTime() time.Duration {
 	taskTimeout := os.Getenv("TASK_HEARTBEAT")
 	if taskTimeout == "" {
-		log.Println("Default timeout: 30s")
-		timeout, _ := time.ParseDuration("30s")
-		return timeout
+		return 0
```

Comment:
Why did you change the default value? This seems to me like it has a high potential to break stuff. 

Heartbeat is used to keep the step function from exiting with an error - https://cl.ly/97bea7f566e1 - and in post processing as well as other step functions we are setting this value - for example https://github.com/Skycatch/post-processing-pipeline-sfn-definition/blob/master/terraform/sfn.tf#L49


And pretty sure that in most workers where we use the heartbeat... we aren't setting a default
- https://github.com/Skycatch/surveyor-groundreturn-worker/blob/master/terraform/docker-ecs.tf
- https://github.com/Skycatch/surveyor-subsample-worker/blob/master/terraform/docker-ecs.tf

Reply:
This function is not used for setting heartbeat anywhere prior to this pr:
https://github.com/Skycatch/tasque-go/blob/master/main.go

ecs handler struct that we're using for jobs has `heartbeatDuration` property. but it not being set, it should not work actually

https://github.com/Skycatch/tasque-go/blob/master/main.go#L104